### DISCUSSION TITLE
finding PA (Propagator Anisotropy)

### DISCUSSION THREAD

**Discussion by maryamafzali** [dipy/dipy#1172] [Reconstruction]
Hi,

I’m interested to find PA (Propagator Anisotropy) using MAP-MRI. I couldn’t find it in your code. Would you please help me? Also I have a question about the estimation of the coefficients using MAP-MRI method (Ozarslan, et al.). Did you consider the constraint on the integral of the probability?

Thank you very much.

Best,
Maryam

**Comment by Garyfallidis** [comment]
@rutgerfick and @maurozucchelli I need your attention in this issue report. @maryamafzali is looking closely to our mapmri implementations and realized that we do not have the PA map. Can you please get back to her and try to resolve the issue?

**Comment by rutgerfick** [comment]
Hi Maryam,

I haven't included the Propagator Anisotropy metric in dipy yet.
I already have it implemented locally, so hopefully I'll have the time to share it with you soon (it's a bit more tricky to implement since you need to combine isotropic and anisotropic fitting of the basis).

The integral contraint on the probability is considered in all cases (using both laplacian and/or positivity constraint).

Let me know if you have any other questions :)

Rutger

**Comment by maryamafzali** [comment]
Hi Rutger,


Thank you very much for your response and all yourexplanations.


Best Regards,

Maryam

**Comment by skoudoro** [comment]
Hi @maryamafzali,

You should look at this discussion [here](https://mail.python.org/pipermail/neuroimaging/2018-January/001617.html) and [here](https://mail.python.org/pipermail/neuroimaging/2018-January/001620.html)

I think it will help you. Thanks to @rutgerfick!
